spec/graphql/relay/relation_connection_spec.rb

```diff
@@ -206,23 +206,82 @@ def get_last_cursor(result)
 
       it "applies to queries by `last`" do
         second_to_last_two_names = ["Death Star", "Shield Generator"]
-        first_and_second_names = ["Yavin", "Echo Base"]
+        first_second_and_third_names = ["Yavin", "Echo Base"]
```

Comment:
There are only two items in this array, why should it be renamed to have `and_third` in the variable name?